### DISCUSSION TITLE
3 colons - what does it mean ?

### DISCUSSION THREAD

**Discussion by Andrzej-Andrzej** [davidgohel/officedown#109] [Q&A]
Hi, 
very basic question as I am beginner, what does it mean:
::: {custom-style="Author Names"} in officedown great package I have seen it.
Where can I read about it or maybe you can point me to some YT materials, thank you. Difficult to find it in google. 
kind regards,
Andrzej

**Comment by davidgohel** [comment]
See https://ardata-fr.github.io/officeverse/office-documents-generation.html#usage-with-rmarkdown

[...]

Being able to use style with paragraphs or text chunks is not a feature of ‘officedown’ but of pandoc.

> From https://pandoc.org/MANUAL.html#custom-styles

[...]
